docs/people-and-groups/managing.md

```diff
@@ -113,6 +113,28 @@ This action will delete any dashboard subscriptions or alerts the person has cre
 
 This action doesn't affect email distribution lists that are managed outside of Metabase.
 
+## Default user accounts
```

Comment:
There are two other things that maybe worth mentioning (unsure)
1. customers are not charged by these accounts
2. it is impossible to log into these accounts